packaging_automation/tests/test_docker_statistics_collector.py

```diff
@@ -0,0 +1,34 @@
+from ..docker_statistics_collector import fetch_and_store_docker_statistics, DockerStats
+
+from datetime import datetime, timedelta
+from ..dbconfig import Base, db_connection_string
+from sqlalchemy import create_engine, text, engine
+from sqlalchemy.orm import sessionmaker
+
+
+def test_docker_statistics_collector():
+    test_day_shift_index = 2
+    test_pull_count_shift = 200
```

Comment:
Maybe we can try something that doesn't divide equally, so we check the modulo part too. (So in this case an odd number)

Reply:
Fixed